config/locales/user_mailer/en.yml

```
@@ -38,6 +38,13 @@ en:
         period, you will receive an email that will ask you to confirm the deletion
         of your %{app} account. Your account will not be deleted until you confirm.'
       subject: How to delete your login.gov account
+    account_verified:
+      contact_link: contact us
+      disavowal_link: change your password
+      intro_html: You successfully verified your identity with %{app} on %{date} using
+        Login.gov. If you did not perform this action, please %{contact_link} and
```

replace this with `%{app_name}`?

```suggestion
        %{app_name}. If you did not perform this action, please %{contact_link} and
```

---

great catch, fixed in 1bf7dc65a74a5e858966954084c8069f75bd7dde